scripts/compile.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Configure to exit script as soon as a command fails.
+set -o errexit
+
+SOLC_05_DIR=solc-0.5
+
+# Delete any previous build artifacts
+rm -rf build/
+
+# Create a subproject where 0.5.x compilation will take place
+mkdir -p $SOLC_05_DIR
+
+cd $SOLC_05_DIR
+npm init --yes
+npm install --save-dev truffle@5.0.0
+
+rm -rf contracts
+ln -s ../contracts contracts
+
+# Delete any previous build artifacts
+rm -rf build/
+
+# Compile
+echo "
+module.exports = {
+  compilers: {
+    solc: {
+      version: \"0.5.0\",
+    },
+  },
+};
+" > truffle-config.js
+
+npx truffle compile
+
+# Modify the paths in the artifacts to make it look as if they were built in the root
+find build -name "*.json" -exec sed -i "s/\/$SOLC_05_DIR//g" {} +
```

Comment:
There's a couple horrible ways this line could fail because of Bash things. But this will be a short-lived script and it works.